CoreVulkanDisplay/src/au/gov/asd/tac/constellation/visual/vulkan/renderables/CVKAxesRenderable.java

```diff
@@ -918,97 +792,21 @@ private void DestroyCommandBuffers() {
-        final int imageCount = cvkSwapChain.GetImageCount();
-        LongBuffer pLayouts = stack.mallocLong(imageCount);
-        for (int i = 0; i < imageCount; ++i) {
-            pLayouts.put(i, hDescriptorLayout);
-        }
-
-        VkDescriptorSetAllocateInfo allocInfo = VkDescriptorSetAllocateInfo.callocStack(stack);
-        allocInfo.sType(VK_STRUCTURE_TYPE_DESCRIPTOR_SET_ALLOCATE_INFO);
-        allocInfo.descriptorPool(cvkDescriptorPool.GetDescriptorPoolHandle());
-        allocInfo.pSetLayouts(pLayouts);            
-
-        // Allocate the descriptor sets from the descriptor pool, they'll be unitialised
-        pDescriptorSets = MemoryUtil.memAllocLong(imageCount);
-        ret = vkAllocateDescriptorSets(cvkDevice.GetDevice(), allocInfo, pDescriptorSets);
-        checkVKret(ret);
-
-        // Struct for the size of the uniform buffer used by PassThru.vs (we fill the actual buffer below)
-        VkDescriptorBufferInfo.Buffer vertBufferInfo = VkDescriptorBufferInfo.callocStack(1, stack);
-        vertBufferInfo.offset(0);
-        vertBufferInfo.range(VertexUniformBufferObject.SIZEOF);      
-
-        // We need 1 write descriptors, 1 for uniform buffers (vs) 
-        VkWriteDescriptorSet.Buffer descriptorWrites = VkWriteDescriptorSet.callocStack(1, stack);
-
-        VkWriteDescriptorSet vertUBDescriptorWrite = descriptorWrites.get(0);
-        vertUBDescriptorWrite.sType(VK_STRUCTURE_TYPE_WRITE_DESCRIPTOR_SET);
-        vertUBDescriptorWrite.dstBinding(0);
-        vertUBDescriptorWrite.dstArrayElement(0);
-        vertUBDescriptorWrite.descriptorType(VK_DESCRIPTOR_TYPE_UNIFORM_BUFFER);
-        vertUBDescriptorWrite.descriptorCount(1);
-        vertUBDescriptorWrite.pBufferInfo(vertBufferInfo);                       
-
-        for (int i = 0; i < imageCount; ++i) {
-            long descriptorSet = pDescriptorSets.get(i);
-
-            vertBufferInfo.buffer(vertexUniformBuffers.get(i).GetBufferHandle());
-            vertUBDescriptorWrite.dstSet(descriptorSet);
-
-            // Update the descriptors with a write and no copy
-            vkUpdateDescriptorSets(cvkDevice.GetDevice(), descriptorWrites, null);
-        }   
-        
-        descriptorPoolResourcesDirty = false;
-        
-        return ret;
-    }       
 
     private int CreateDescriptorPoolResources() {
         CVKAssert(cvkDescriptorPool != null);
         CVKAssert(cvkSwapChain != null);
-        try (MemoryStack stack = stackPush()) {
-            return CreateDescriptorSets(stack);
-        }
+        return VK_SUCCESS;
     }     
     
     @Override
     public void IncrementDescriptorTypeRequirements(CVKDescriptorPoolRequirements reqs, CVKDescriptorPoolRequirements perImageReqs) {
-        // PassThru.vs
-        ++perImageReqs.poolDescriptorTypeCounts[VK_DESCRIPTOR_TYPE_UNIFORM_BUFFER];
-        
-        // One set per image
-        ++perImageReqs.poolDesciptorSetCount;        
+        // No descriptor sets required
```

Comment:
...because axes use push constants instead of descriptor bound uniform buffers.

Reply:
Modified comment